.github/workflows/release-snapshot.yml

```diff
@@ -42,6 +42,20 @@ jobs:
         key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
         restore-keys: |
           ${{ runner.os }}-yarn-
+    - name: Initialize babel cache
+      id: babel-cache
+      uses: actions/cache@v2
+      with:
+        path: .cache/babel
+        key: ${{ runner.os }}-babel-cache-${{ hashFiles('config/babel.config.json', 'config/webpack.config.babel.js') }}
+        restore-keys: |
+          ${{ runner.os }}-babel-cache-
+    - name: Initialize public folder cache
```

Comment:
Would it make sense to move this step a little closer to where it's actually used? So above `name: Build frontend`?